scripts/ansible/ansible.cfg

```diff
@@ -1,6 +1,16 @@
 [defaults]
+any_errors_fatal = true
 transport = ssh
 deprecation_warnings = false
+host_key_checking = false
```

Comment:
Do we really want this at the global config level?

Reply:
I'm happy to remove it, but note I haven't changed the setting from what is master (as `ssh_args = ... o StrictHostKeyChecking=no`), merely the method by which it's configured.

Reply:
You're right.